README.md

```diff
@@ -64,7 +62,8 @@ extension IPOrigin: JSONMappable {
     }
 }
 
-let request = NetworkRequest(path: "/ip", baseURLKey: baseURLKey)
+let url = URL(string: "https://httpbin.org/ip")!
```

Comment:
There is a custom initializer provided by us ;)

https://github.com/dbsystel/DBNetworkStack/blob/develop/Source/URLRequestConvertible.swift#L45